iree/testing/benchmark_full.cc

```diff
@@ -111,7 +111,7 @@ static void iree_benchmark_run(const char* benchmark_name,
 }
 
 void iree_benchmark_register(iree_string_view_t name,
-                             const iree_benchmark_def_t* benchmark_def) {
+                             iree_benchmark_def_t* benchmark_def) {
```

Comment:
I think we don't want the object pointed by `benchmark_def`, so we should keep `const`?